domains/worm.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "arq091",
+        "email": "poonyapat.thane@gmail.com"
+      },
+      "record": {
+        "CNAME": "arq091.github.io/wiggly-worm"
```

```suggestion
        "CNAME": "arq091.github.io"
```

The wriggly worm website will still show up even when the `CNAME` doesn't have the URL.